tsconfig.base.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2018",
+    "target": "esnext",
```

Comment:
Lol, haven't had `master` pulled in `glow-js` when I checked so I missed https://github.com/glow-xyz/glow-js/pull/63. Will see what I can do!

Reply:
Yea, so I went to `es6` since there were issues with `esnext` and `es2020` when I tried to import that code into Solana Explorer which uses create-react-app (that has some old babel stuff).

Reply:
I don't fully get why they (or we) would set `target` to `es5` (`es6`) and add `esnext` to `lib`… The way I understand it (and I may be wrong) is that:
- TypeScript will let us use new features like `Object.values`, `[].includes`, `BigInt` (thanks to `esnext` in `lib`)
- but will try to compile modern structures like classes to older style code.

Which is what one would do if they knew the environment that the code will be run on supports some modern features (hence `lib`). But if we suspect the environment supports `esnext`, why not compile to `esnext`? Haha

Changed configuration around as suggested! [`0053451` (#3)](https://github.com/glow-xyz/beet/pull/3/commits/0053451da859efde0f8de11573e2db5eba5c3c1f)

<img width="850" alt="Zrzut ekranu 2022-07-22 o 18 14 58" src="https://user-images.githubusercontent.com/1151041/180480947-8ecef17f-f0e2-449b-ab2c-acaaf59e9299.png">